### DISCUSSION TITLE
Does this package support write sas7bdat？

### DISCUSSION THREAD

**Discussion by zhangpanpan01** [Roche/pyreadstat#263] [Q&A]
Does this package support write sas7bdat？

**Comment by ofajardo** [comment]
Unfortunately not at the current moment. You can save the data as SAS xport and then read it in SAS transforming it to sas7bdat.

The reason is that the underlying C library does allow to save sas7bdat files, but those files cannot be read from SAS, SAS raises an error, and therefore there are not very useful. Once the C library, eventually in the future, does write sas7bdat files that are readable from SAS, the feature will be implemented here.
